### PR TITLE
Fix: hooks in all reown AppKit Docs

### PR DESCRIPTION
**Pull request by rtomas**
+ Add the useAppKitAccount hook to react and next docs
+ Fix all hooks docs

### CODE REVIEW

File: docs/appkit/next/solana/hooks.mdx [hunk 1]
```diff
@@ -51,7 +51,7 @@ function SignMessage() {
 ### useAppKitError
 
 ```ts
-import { useAppKitError } from '@web3modal/solana/react'
+import { useAppKitError } from '@reown/solana/react'
```
Comment (rohit-710) [inline docs/appkit/next/solana/hooks.mdx:54]: I don't think this is the correct import structure. Can you please check if this works locally with your code?

File: docs/appkit/next/solana/hooks.mdx [hunk 2]
```diff
@@ -5,7 +5,7 @@ import Button from '../../../components/button'
 Hook that returns the client's information.
 
 ```tsx
-import { useAppKitAccount } from '@web3modal/solana/react'
+import { useAppKitAccount } from '@reown/solana/react'
```
Comment (rohit-710) [inline docs/appkit/next/solana/hooks.mdx:8]:
```suggestion
import { useAppKitAccount } from '@reown/appkit/react'
```

I think this should be the correct package.

File: docs/appkit/next/solana/about/programs.mdx [hunk 3]
```diff
@@ -23,7 +23,7 @@ function deserializeCounterAccount(data?: Buffer): { count: number } {
   }
 }
 
-const { address, currentChain } = useAppKitAccount()
+const { address } = useAppKitAccount()
```
Comment (rohit-710) [inline docs/appkit/next/solana/about/programs.mdx:26]: Have you tested this out?

File: docs/appkit/react/core/custom-connectors.mdx [hunk 4]
```diff
@@ -71,7 +71,7 @@ export function ContextProvider({ children }) {
 }
 ```
 
-Check our React Wagmi demo in [Github](https://github.com/WalletConnect/web-examples/tree/main/dapps/web3modal/react)
+Check our React Wagmi demo in [Github](https://github.com/WalletConnect/web-examples/tree/main/dapps/)
```
Comment (rohit-710) [inline docs/appkit/react/core/custom-connectors.mdx:74]:
```suggestion
Check our React Wagmi demo in [Github](https://github.com/reown-com/web-examples/tree/main/dapps/web3modal/react)
```

Shouldn't this be the correct URL?

File: docs/appkit/react/core/custom-connectors.mdx [hunk 5]
```diff
@@ -111,7 +111,7 @@ ReactDOM.createRoot(document.getElementById('root') as HTMLElement).render(
 )
 ```
 
-Check our React ethers demo in [Github](https://github.com/WalletConnect/web-examples/tree/main/dapps/web3modal/react-ethers)
+Check our React ethers demo in [Github](https://github.com/WalletConnect/web-examples/tree/main/dapps/)
```
Comment (rohit-710) [inline docs/appkit/react/core/custom-connectors.mdx:114]:
```suggestion
Check our React ethers demo in [Github](https://github.com/reown-com/web-examples/tree/main/dapps/web3modal/react-ethers)
```

I think this is the correct URL, please confirm!

File: docs/appkit/react/ethers/hooks.mdx [hunk 6]
```diff
@@ -1,19 +1,5 @@
...
-```tsx
-import { useAppKitAccount } from '@reown/base/react'
-
-function Components() {
-  const { address, status, isConnected } = useAppKitAccount()
-
-  //...
-}
-```
-
```
Comment (rohit-710) [inline docs/appkit/react/ethers/hooks.mdx:16]: This hook exists on `@reown/appkit/react`. Any particular reason why we are removing this? 

It also exists on the latest `1.1.0` version of AppKit. Can you please confirm and add it back?

File: docs/appkit/react/solana/implementation.mdx [hunk 7]
```diff
@@ -4,7 +4,7 @@ import TabItem from '@theme/TabItem'
 On top of your app set up the following configuration, making sure that all functions are called outside any React component to avoid unwanted rerenders.
 
 ```tsx
-import { createAppKit, defaultConfig } from '@web3modal/solana/react'
+import { createAppKit, defaultConfig } from '@reown/solana/react'
```
Comment (rohit-710) [inline docs/appkit/react/solana/implementation.mdx:7]:
```suggestion
import { createAppKit } from '@reown/appkit/react'
```

You can only import `createAppKit` from `@reown/appkit` or `reown/appkit/react`. I am unable to find `defaultConfig` inside this package. Can you please check?

File: docs/appkit/react/solana/triggermodal.mdx [hunk 8]
```diff
@@ -24,7 +24,7 @@ Web components are global html elements that don't require importing.
 You can trigger the modal by calling the `open` function from `useAppKit` hook.
 
 ```tsx
-import { useAppKit } from '@web3modal/ethers5/react'
+import { useAppKit } from '@reown/ethers5/react'
```
Comment (rohit-710) [inline docs/appkit/react/solana/triggermodal.mdx:27]:
```suggestion
import { useAppKit } from '@reown/appkit/react'
```

`@reown/ethers5` does not exist

File: docs/appkit/react/wagmi/hooks.mdx [hunk 9]
```diff
@@ -2,22 +2,6 @@ import Button from '../../../components/button'
...
-
-function App() {
-  const { address, isConnecting, isDisconnected } = useAccount()
-
-  if (isConnecting) return <div>Connecting…</div>
-  if (isDisconnected) return <div>Disconnected</div>
-  return <div>{address}</div>
-}
-```
-
```
Comment (rohit-710) [inline docs/appkit/react/wagmi/hooks.mdx:20]: Can't we still use this Wagmi function? 

This should still work (I tested it and can confirm that it works), can you please confirm? I don't think we should remove this from the docs.

File: docs/appkit/vue/core/composables.mdx [hunk 10]
```diff
@@ -4,133 +4,75 @@ import PlatformTabItem from '../../../components/PlatformTabItem'
...
 
-## useAppKit
+## useAppKitAccount
 
-Control the modal with the `useAppKit` function
+Hook for accessing account data and connection status.
 
-<PlatformTabs groupId="eth-lib" activeOptions={["wagmi", "ethers5","ethers", "solana"]}>
-<PlatformTabItem value="wagmi">
+List of possible `status`: "reconnecting" | "connected" | "disconnected" | "connecting" | undefined
```
Comment (rohit-710) [inline docs/appkit/vue/core/composables.mdx:17]:
```suggestion
List of possible `status: "reconnecting" | "connected" | "disconnected" | "connecting" | undefined`
```

File: docs/appkit/vue/core/composables.mdx [hunk 11]
```diff
@@ -4,133 +4,75 @@ import PlatformTabItem from '../../../components/PlatformTabItem'
...
 
-```ts
-import { useAppKit } from '@reown/appkit/vue'
+The `caipAddress` is composed by `network:chainId:address`. An Example for an Ethereum mainnet address is eip155:1:0xab16a96D359eC26a11e2C2b3d8f8B8942d5Bfcdb
```
Comment (rohit-710) [inline docs/appkit/vue/core/composables.mdx:19]:
```suggestion
The `caipAddress` is composed by `network:chainId:address`. An Example for an Ethereum mainnet address is `eip155:1:0xab16a96D359eC26a11e2C2b3d8f8B8942d5Bfcdb`
```

File: docs/appkit/vue/core/composables.mdx [hunk 12]
```diff
@@ -4,133 +4,75 @@ import PlatformTabItem from '../../../components/PlatformTabItem'
...
 
-</PlatformTabItem>
-<PlatformTabItem value="ethers5">
+## useAppKit
+
+Control the modal with the `useAppKit` hook
 
 ```ts
-import { useAppKit } from '@reown/appkit/vue'
+import { useAppKit } from "@reown/appkit/vue";
```
Comment (rohit-710) [inline docs/appkit/vue/core/composables.mdx:34]:
```suggestion
import { useAppKit } from '@reown/appkit/vue';
```

File: docs/appkit/vue/ethers5/triggermodal.mdx [hunk 13]
```diff
@@ -25,7 +25,7 @@ You can trigger the modal by calling the `open` method from `useAppKit` composab
 
 ```html
 <script setup lang="ts">
-  import { useAppKit } from '@web3modal/ethers5/vue'
+  import { useAppKit } from '@reown/ethers5/vue'
```
Comment (rohit-710) [inline docs/appkit/vue/ethers5/triggermodal.mdx:28]:
```suggestion
  import { useAppKit } from '@reown/appkit/vue'
```

`@reown/ethers5` does not exist

File: docs/appkit/vue/solana/about/implementation.mdx [hunk 14]
```diff
@@ -5,8 +5,8 @@ In your `App.vue` file set up the following configuration
 ```html
 <script setup>
   
-  import { createAppKit, defaultSolanaConfig } from '@web3modal/solana/vue'
-  import { solana, solanaTestnet, solanaDevnet } from '@web3modal/solana/chains'
+  import { createAppKit, defaultSolanaConfig } from '@reown/solana/vue'
+  import { solana, solanaTestnet, solanaDevnet } from '@reown/solana/chains'
```
Comment (rohit-710) [inline docs/appkit/vue/solana/about/implementation.mdx:9]:
```suggestion
  import { createAppKit } from '@reown/appkit/vue'
  import { solana, solanaTestnet, solanaDevnet } from '@reown/appkit/networks'
```

`defaultSolanaConfig` is no longer required with AppKit. The Solana networks can be imported from `@reown/appkit/networks`

File: docs/appkit/react/core/hooks.mdx [hunk 15]
```diff
@@ -4,136 +4,75 @@ import PlatformTabItem from '../../../components/PlatformTabItem'
...
 
-</PlatformTabItem>
-<PlatformTabItem value="ethers5">
+## useAppKit
+
+Control the modal with the `useAppKit` hook
 
 ```ts
-import { useAppKit } from '@reown/appkit/react'
+import { useAppKit } from "@reown/appkit/react";
```
Comment (rohit-710) [inline docs/appkit/react/core/hooks.mdx:35]:
```suggestion
import { useAppKit } from '@reown/appkit/react';
```

File: docs/appkit/react/core/hooks.mdx [hunk 16]
```diff
@@ -4,136 +4,75 @@ import PlatformTabItem from '../../../components/PlatformTabItem'
...
 
-Control the modal with the `useAppKit` hook
-
-<PlatformTabs groupId="eth-lib" activeOptions={["wagmi", "ethers5","ethers", "solana"]}>
-<PlatformTabItem value="wagmi">
+Hook for accessing account data and connection status.
 
-```ts
-import { useAppKit } from '@reown/appkit/react'
+List of possible `status`: "reconnecting" | "connected" | "disconnected" | "connecting" | undefined
```
Comment (rohit-710) [inline docs/appkit/react/core/hooks.mdx:18]:
```suggestion
List of possible `status: "reconnecting" | "connected" | "disconnected" | "connecting" | undefined`
```

File: docs/appkit/react/core/hooks.mdx [hunk 17]
```diff
@@ -4,136 +4,75 @@ import PlatformTabItem from '../../../components/PlatformTabItem'
...
 
-export default function Component() {
-  const { open, close } = useAppKit()
+The `caipAddress` is composed by `network:chainId:address`. An Example for an Ethereum mainnet address is eip155:1:0xab16a96D359eC26a11e2C2b3d8f8B8942d5Bfcdb
```
Comment (rohit-710) [inline docs/appkit/react/core/hooks.mdx:20]:
```suggestion
The `caipAddress` is composed by `network:chainId:address`. An Example for an Ethereum mainnet address is `eip155:1:0xab16a96D359eC26a11e2C2b3d8f8B8942d5Bfcdb`
```

File: docs/appkit/next/core/hooks.mdx [hunk 18]
```diff
@@ -5,133 +5,74 @@ import PlatformTabItem from '../../../components/PlatformTabItem'
...
+## useAppKitAccount
 
-Control the modal with the `useAppKit` hook
+Hook for accessing account data and connection status.
 
-<PlatformTabs groupId="eth-lib" activeOptions={["wagmi", "ethers5","ethers","solana"]}>
+List of possible `status`: "reconnecting" | "connected" | "disconnected" | "connecting" | undefined
 
-<PlatformTabItem value="wagmi">
+The `caipAddress` is composed by `network:chainId:address`. An Example for an Ethereum mainnet address is eip155:1:0xab16a96D359eC26a11e2C2b3d8f8B8942d5Bfcdb
```
Comment (rohit-710) [inline docs/appkit/next/core/hooks.mdx:20]:
```suggestion
The `caipAddress` is composed by `network:chainId:address`. An Example for an Ethereum mainnet address is `eip155:1:0xab16a96D359eC26a11e2C2b3d8f8B8942d5Bfcdb`
```

File: docs/appkit/next/solana/about/programs.mdx (hunk 3 above)
Comment (rtomas) [reply docs/appkit/next/solana/about/programs.mdx:26]: https://github.com/reown-com/web-examples/pull/729

File: docs/appkit/react/solana/hooks.mdx [hunk 19]
```diff
@@ -19,7 +19,7 @@ function Components() {
 Hook that returns the `walletProvider` and the `WalletProviderType`.
 
 ```tsx
-import { useAppKitAccount, useAppKitrovider } from '@web3modal/solana/react'
+import { useAppKitAccount, useAppKitrovider } from '@reown/solana/react'
```
Comment (rohit-710) [inline docs/appkit/react/solana/hooks.mdx:22]:
```suggestion
import { useAppKitAccount, useAppKitrovider } from '@reown/appkit/react'
```

**Review by rohit-710 — APPROVED** [review]

**Pull request merged**